db/schema.rb

```diff
@@ -799,7 +802,7 @@
     t.integer "invited_by_id"
     t.integer "invitations_count", default: 0
     t.boolean "organization_admin"
-    t.string "name", default: "Name Not Provided", null: false
+    t.string "name"
```

Comment:
Not entirely clear how this got removed. I don't see anything that is explicitly removing the default. Unless you are intentionally removing this, this shouldn't be modified.

Reply:
I will revert this line. It is a line that keeps appearing because I had to change it in one of my PRs that is not merged yet. It shouldn't be here.

Reply:
It could be that this affects the test. I will run everything to double-check on that.